### PR TITLE
Views

### PR DESCRIPTION
**Pull request by joyapisi**
## Work Done
- Use preprocessed HTML file with embedded Ruby code.
- Use layouts and templates for shared content.

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1 +1,7 @@
-<h1>Users here</h1>
\ No newline at end of file
+<h1>Users here</h1>
+
+<% @users.each do |user| %>
+
+    <%= render 'shared/users', user: user %>
+    
+<% end %>
```
Comment (VICTOR-CHUKWU) [inline app/views/users/index.html.erb]: - [ ] kindly match the given wireframe 
your design :point_down: 
Screenshot from 2023-07-20 22-18-20
wireframe :point_down: 
Screenshot from 2023-07-20 22-19-45

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,19 @@
...
+    <div>
+        <h1>post <%= post.title %></h1>
+        <p><%= post.text %></p>
+        <p>Comments: <%= post.comments_counter %> Likes: <%= post.likes_counter %></p>
+    </div>
+<%end%>
+    
+
+<a href="/users/<%= @user.id%>/posts" >See all Posts</a>
+</div>
```
Comment (VICTOR-CHUKWU) [inline app/views/users/show.html.erb]: - [ ] kindly match the wireframe given 
your design :point_down: 
Screenshot from 2023-07-20 22-23-31
wireframe :point_down: 
Screenshot from 2023-07-20 22-23-13

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1 +1,20 @@
...
+<div>
+<% Post.recent_comments(post).each  do |comment|%>
+    <p><%= comment.author.name%> - <%= comment.text %></p>
+    <%end%>
+</div>
+<br />
+<%end%>
+<div id="bottom-button">
+<button>Pagination</button>
+</div>
```
Comment (VICTOR-CHUKWU) [inline app/views/posts/index.html.erb]: - [ ] When I navigate to the post page of a user posts page I get an error
Screenshot from 2023-07-20 22-25-59

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @joyapisi  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — COMMENTED** [review]
Hi @joyapisi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has the wrong link for the activity since you submitted it in add form activity and the views milestone so please kae sure you submit it in the right activity ✨💪🏻🙏🏻🤩

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED :1st_place_medal: 

Hi @joyapisi , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Requested changes implemented

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**